examples/example_learn_samples.py

```diff
@@ -0,0 +1,67 @@
+# %%
+"""
+=============================
+Density Compensation Routines
+=============================
+
+Examples of differents density compensation methods.
+
+Density compensation depends on the sampling trajectory,and is apply before the
+adjoint operation to act as preconditioner, and should make the lipschitz constant
+of the operator roughly equal to 1.
+
+"""
+import brainweb_dl as bwdl
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+
+from mrinufft import check_backend, get_density, get_operator
+from mrinufft.trajectories import initialize_2D_radial
+from mrinufft.trajectories.display import display_2D_trajectory
+
+# %%
+# Create sample data
+# ------------------
+
+class Model(torch.nn.Module):
+    def __init__(self):
```

Comment:
Why do you need the extra module layer ?

Reply:
I think and expect this is how users will use our operator. The `test` handles the case when they dont instance in a module and the example handles it being instanced in a module. Note that this was failing earlier, and hence we have this PR to begin with. Eventually we might want the model to dynamically update the shape of the trajectory (during upsampling steps for example.